src/installer/install.ts

```diff
@@ -23,8 +25,9 @@ export async function retreiveFromCache(file: ClientFile): Promise<string | unde
         + "For more information, see https://github.com/actions/cache/issues/505");
     }
 
-    if (process.env[SKIP_CACHE_ENVVAR] === "true") {
-        ghCore.info(`⏩ ${SKIP_CACHE_ENVVAR} is set; skipping cache check.`);
+    if (process.env[SKIP_CACHE_ENVVAR] === "true" || skipCache === "true") {
```

Comment:
i feel like all the 'skip cache' logic (ie env set, input set, OR is GHES) should be encapsulated into a function like

`export function shouldUseCache(): boolean`